src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/entity/HdcVariationLicence.kt

```diff
@@ -0,0 +1,511 @@
+  dateOfBirth = dateOfBirth,
+  conditionalReleaseDate = conditionalReleaseDate,
+  actualReleaseDate = actualReleaseDate,
+  sentenceStartDate = sentenceStartDate,
+  sentenceEndDate = sentenceEndDate,
+  licenceStartDate = licenceStartDate,
+  licenceExpiryDate = licenceExpiryDate,
+  licenceActivatedDate = licenceActivatedDate,
+  topupSupervisionStartDate = topupSupervisionStartDate,
+  topupSupervisionExpiryDate = topupSupervisionExpiryDate,
+  postRecallReleaseDate = postRecallReleaseDate,
+  probationAreaCode = probationAreaCode,
+  probationAreaDescription = probationAreaDescription,
+  probationPduCode = probationPduCode,
+  probationPduDescription = probationPduDescription,
+  probationLauCode = probationLauCode,
+  probationLauDescription = probationLauDescription,
+  probationTeamCode = probationTeamCode,
+  probationTeamDescription = probationTeamDescription,
+  appointmentPersonType = appointmentPersonType,
+  appointmentPerson = appointmentPerson,
+  appointmentTime = appointmentTime,
+  appointmentTimeType = appointmentTimeType,
+  appointmentAddress = appointmentAddress,
+  appointmentContact = appointmentContact,
+  approvedDate = approvedDate,
+  approvedByUsername = approvedByUsername,
+  approvedByName = approvedByName,
+  supersededDate = supersededDate,
+  submittedDate = submittedDate,
+  dateCreated = dateCreated,
+  dateLastUpdated = dateLastUpdated,
+  updatedByUsername = updatedByUsername,
+  licenceVersion = licenceVersion,
+  standardConditions = standardConditions,
+  additionalConditions = additionalConditions,
+  bespokeConditions = bespokeConditions,
+  responsibleCom = responsibleCom,
+  updatedBy = updatedBy,
+) {
+
+  fun copy(
+    id: Long = this.id,
+    typeCode: LicenceType = this.typeCode,
+    version: String? = this.version,
+    statusCode: LicenceStatus = this.statusCode,
+    nomsId: String? = this.nomsId,
+    bookingNo: String? = this.bookingNo,
+    bookingId: Long? = this.bookingId,
+    crn: String? = this.crn,
+    pnc: String? = this.pnc,
+    cro: String? = this.cro,
+    prisonCode: String? = this.prisonCode,
+    prisonDescription: String? = this.prisonDescription,
+    prisonTelephone: String? = this.prisonTelephone,
+    forename: String? = this.forename,
+    middleNames: String? = this.middleNames,
+    surname: String? = this.surname,
+    dateOfBirth: LocalDate? = this.dateOfBirth,
+    conditionalReleaseDate: LocalDate? = this.conditionalReleaseDate,
+    actualReleaseDate: LocalDate? = this.actualReleaseDate,
+    sentenceStartDate: LocalDate? = this.sentenceStartDate,
+    sentenceEndDate: LocalDate? = this.sentenceEndDate,
+    licenceStartDate: LocalDate? = this.licenceStartDate,
+    licenceExpiryDate: LocalDate? = this.licenceExpiryDate,
+    licenceActivatedDate: LocalDateTime? = this.licenceActivatedDate,
+    homeDetentionCurfewActualDate: LocalDate? = this.homeDetentionCurfewActualDate,
+    homeDetentionCurfewEndDate: LocalDate? = this.homeDetentionCurfewEndDate,
```

Comment:
`homeDetentionCurfewActualDate` and `homeDetentionCurfewEndDate` = HDC specific fields